dwave_networkx/topology.py

```diff
@@ -0,0 +1,77 @@
+from enum import Enum, auto
+
+try:
+    from enum import global_enum
+except ImportError:
+    # delete after Py3.10 EOL (2026/10)
+    import sys
+
+    def global_enum(cls):
```

Comment:
Might also be worth a comment that this implementation is backported from the Python enum library